<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NuevaPublicacion</class>
 <widget class="QDockWidget" name="NuevaPublicacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Crear Publiacion</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_entradaPublicacion">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>571</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_publicar">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>410</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Publicar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_regresar">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>410</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Regresar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_imagen">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>410</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar imagen</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
